### DISCUSSION TITLE
How to avoid triggering Metadata queries

### DISCUSSION THREAD

**Discussion by subbareddydagumati** [dbeaver/dbeaver#14465] [Q&A]
#### System information: 
- Windows 10
- DBeaver 7.3.1.202012201616

#### Connection specification:
- PrestoSql-346
- PrestoSql Jdbc - 339
- Do you use tunnels or proxies (SSH, SOCKS, etc)?  N

#### Describe the problem you're observing:
DBeaver automatically triggering metadata queries in the background like below. This causing too many active queries on server and they running for longer duration. How to avoid triggering these unwanted metadata queries

SELECT TABLE_CAT, TABLE_SCHEM, TABLE_NAME, TABLE_TYPE, REMARKS,
  TYPE_CAT, TYPE_SCHEM, TYPE_NAME,   SELF_REFERENCING_COL_NAME, REF_GENERATION
FROM system.jdbc.tables
WHERE TABLE_CAT = 'dl_prod' AND TABLE_NAME LIKE 'rep' ESCAPE '\'
ORDER BY TABLE_TYPE, TABLE_CAT, TABLE_SCHEM, TABLE_NAME

SELECT TABLE_CAT, TABLE_SCHEM, TABLE_NAME, TABLE_TYPE, REMARKS,
  TYPE_CAT, TYPE_SCHEM, TYPE_NAME,   SELF_REFERENCING_COL_NAME, REF_GENERATION
FROM system.jdbc.tables
WHERE TABLE_CAT = 'dl_prod' AND TABLE_NAME LIKE 'repository' ESCAPE '\'
ORDER BY TABLE_TYPE, TABLE_CAT, TABLE_SCHEM, TABLE_NAME

**Comment by kseniiaguzeeva** [comment]
Similar issues #9875, #9377

**Comment by uslss** [comment]
you could try to uncheck Open separate connection for metadata read in Preferences
![image](https://user-images.githubusercontent.com/31996417/139814039-0e7ef8ec-4cb3-426c-b99d-c5cee2a78434.png)

**Reply by ldsalomone** [reply]
@uslss is there a way to disable these metadata queries all together for a single connection? The above checkbox does not help because while there might be a separate connection presto still groups the queries by user and only allows 2 queries at a time by default. sometime dbeaver fires off two metadata queries and then hang the actual query in the queue until the metadata completes.
